OJP/OJP_FareSupport.xsd

```diff
@@ -556,6 +556,11 @@
 				</xs:annotation>
 			</xs:element>
 			<xs:group ref="FareProductPriceGroup"/>
+			<xs:element name="AvailableTicketsAtPrice" type="xs:positiveInteger" minOccurs="0">
```

Comment:
The Name is FareQuota in Transmodel/NeTEx
![image](https://github.com/VDVde/OJP/assets/3328999/1fc0c18d-f3bd-4682-8078-0e9b7f1364d2)